lib/Ed25519KeyPair.js

```diff
@@ -11,32 +11,72 @@ const LDKeyPair = require('./LDKeyPair');
 
 class Ed25519KeyPair extends LDKeyPair {
   /**
-   * @param options {object} See LDKeyPair constructor docstring for full list
-   *
-   * @param [options.publicKeyBase58] {string}
-   * @param [options.privateKeyBase58] {string}
+   * @classdesc  An implementation of
+   * [Ed25519 Signature 2018]{@link https://w3c-dvcg.github.io/lds-ed25519-2018/}
+   * for
+   * [jsonld-signatures.]{@link https://github.com/digitalbazaar/jsonld-signatures}
+   * @description Creates an [Ed25519]{@link https://ed25519.cr.yp.to/}
+   * Key Pair usable by a json-ld signature.
+   * @example
+   * const ops = {
+   *   publicKeyBase58: 'H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV',
+   *   privateKeyBase58: 'K3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV'
```

Comment:
Real private keys are bigger than the public keys and not almost the same.  Just to avoid confusion, maybe use some different base58 junk with some ellipsis?

Reply:
- [ ] Change private key to a bigger base58 string with some ellipsis.

Reply:
@dlongley  you mean like 'PRIVATEKEYEXAMPLEDONOTUSESFSFSDF...'

Reply:
I'm just trying to make it obvious to readers.  I think private key is 2x public key length?  Your new choice is the very unlikely base58 value that happens to be uppercase base16.  I hope that doesn't confuse anyone.  I suggested ellipsis to just make the text shorter.  Could also just generate real values so code examples actually work.

Reply:
I used this: https://www.browserling.com/tools/base58-encode

and generated a smaller publicKey and a twice as long private key.

Reply:
@davidlehn the keys in this file are Base58 encoded and small enough to be practical for examples. I also checked and they do work as in you can create a KeyPair with them, generates a fingerprint, verifyFingerPrints etc. so not the exact solution suggested, but either better or reveals that there needs to be a check for strength length when generating key pairs. @mattcollier short strings are fine. So mark as resolved or make an issue we need to ensure key length is a certain size when generating a KeyPair.